src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -274,33 +279,341 @@
         expect(bs_request.reload.state).to eq(:revoked)
       end
 
-      it 'adds submitter as maintainer' do
-        login(receiver)
-        post :changerequest, params: {
-          number: bs_request.number, accepted: 'accepted',
-          add_submitter_as_maintainer_0: "#{target_project}_#_#{target_package}" # rubocop:disable Naming/VariableNumber
-        }
-        expect(bs_request.reload.state).to eq(:accepted)
-        expect(target_package.relationships.map(&:user_id)).to include(submitter.id)
+      context 'when using the legacy request show page' do
```

Comment:
This isn’t needed because the new version is expected to roll out soon hopefully. So we can likely skip adding anything new to the legacy UI.